src/main/java/chess/domain/position/Lettering.java

```diff
@@ -0,0 +1,84 @@
+package chess.domain.position;
+
+import java.util.Arrays;
+
+public enum Lettering {
+
+    A,
+    B,
+    C,
+    D,
+    E,
+    F,
+    G,
+    H;
+
```

Comment:
다음 enum이 추가될 때를 생각해서 다음과 같이 쓰기도 한답니다 :)
```suggestion
    G,
    H,
    ;
```